YIF_Backend/Controllers/InstitutionOfEducationController.cs

```diff
@@ -211,5 +211,22 @@ public async Task<IActionResult> DeleteInstitutionOfEducationFromFavorite(string
             var result = await _institutionOfEducationService.GetInstitutionOfEducationAbbreviations(filterModel);
             return Ok(result);
         }
+
+        /// <summary>
+        /// Get all directions and specialties of institution of education
+        /// </summary>
+        /// <response code="200">Get all directions and specialties in institution of education</response>
+        /// <response code="404">If there are no directions</response>
```

Comment:
I think that you will have 404 in other situtions (IoEAdmin/ModeratorNotFound, IoENotFound), but not for this case